runtime/src/serde_snapshot.rs

```diff
@@ -97,6 +104,8 @@ impl<T> SnapshotAccountsDbFields<T> {
                 incremental_snapshot_version,
                 incremental_snapshot_slot,
                 incremental_snapshot_bank_hash_info,
+                incremental_snapshot_prior_roots,
+                incremental_snapshot_prior_roots_with_hash,
```

Comment:
nit: Can you update this function's comment to mention the `prior_roots` and `prior_roots_with_hash` fields too?
And since there are more fields now, maybe something like:

```rust
    /// [...] Otherwise use the AccountsDbFields from the increment snapshot, and a combination
    /// of the storages from both the full and incremental snapshots.
```

P.S. I wanted to put this nit *on* the fn's comment itself, but Github would not let me.